apps/ejabberd/priv/cassandra.cql

```diff
@@ -1,4 +1,77 @@
-DROP KEYSPACE IF EXISTS mongooseim;
 CREATE KEYSPACE mongooseim
          WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor': 3};
 USE mongooseim;
+
+CREATE TABLE mam_message(
```

Comment:
How does the MAM related schema compares to what we had in version 1.6.2? Is it compatible?